src/components/TextField/TextField.tsx

```diff
@@ -2,10 +2,10 @@ import clsx from 'clsx';
 import React, { ChangeEventHandler, MouseEventHandler, ReactNode } from 'react';
 import { useUID } from 'react-uid';
 import styles from './TextField.module.css';
-import Icon, { IconName } from '../../components/Icon';
-import FieldNote from '../FieldNote';
-import Label from '../Label';
-import TextInput from '../TextInput';
+import FieldNote from '../../upcoming-components/FieldNote';
+import Label from '../../upcoming-components/Label';
```

Comment:
Let's add `Label` to components as well so this ships with the system

Reply:
Added

Reply:
Added